apps/studio/src/components/tableview/TableTable.vue

```diff
@@ -974,14 +975,20 @@ export default Vue.extend({
         disabled: areAllCellsPrimarykey || !this.editable,
       }
     },
+    isEditorMenuDisabled (cell: CellComponent) {
+      if (this.isPrimaryKey(cell.getField())) return true
+      return !this.editable && !this.insertionCellCheck(cell)
+    },
+    openEditorMenuByShortcut() {
+      const range: RangeComponent = _.last(this.tabulator.getRanges())
+      const cell = range.getCells().flat()[0];
+      if (this.isEditorMenuDisabled(cell)) return
+      this.$refs.editorModal.openModal(cell.getValue(), undefined, cell)
```

Comment:
Would you be able to add a FIXME note here so we can avoid calling child methods directly like this? Normally, it should be done by calling an event using `this.$modal.show(modalName)` or `this.$trigger(AppEvent.something)`.